test/Microsoft.Identity.Abstractions.AotTests/Microsoft.Identity.Abstractions.AotTests.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net8.0</TargetFrameworks>
```

Comment:
you should be able to use the Directory.Build.Prop file in the test section.

Reply:
I'm testing AoT. so only 8 and 9